plugins/inputs/ecs/types.go

```diff
@@ -0,0 +1,81 @@
+package ecs
+
+import (
+	"encoding/json"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+)
+
+// Task is the ECS task representation
+type Task struct {
+	Cluster       string
+	TaskARN       string
+	Family        string
+	Revision      string
+	DesiredStatus string
+	KnownStatus   string
+	Containers    []Container
+	Limits        map[string]float64
+	PullStartedAt time.Time
```

Comment:
Is it possible to have a time.Time as a field with json.Unmarshal?  How does it know how to parse the time?  Please check if this is working I suspect it is not.

Reply:
it works. as long as the time string is fully ISO8601/RFC3339 (which it always is here).
https://play.golang.org/p/KgxNP7hKkMV